src/main/resources/templates/template-ls.json

```diff
@@ -34,6 +34,20 @@
         }
       ]
     },
+    {
+      "name": "TypeScript Language Server",
+      "dev": true,
+      "docPath": "node/typescript-language-server.md",
+      "programArgs": {
+        "default": "typescript-language-server --stdio"
+      },
+      "fileTypeMappings": [
+        {
+          "fileType": "JavaScript",
+          "languageId": "javascript"
+        }

```

Comment:
what if you add TypeScript in the template, but it's not available (yet) in the IDE?

Reply:
The mapping is ignored in this case. But we need to improve it:

 * ask to the user to create the file type 
 * if user says "no", it should be add as file patterns

Reply:
so just add TypeScript for now

Reply:
added